src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -49,6 +51,10 @@
     @Nullable
     private EventTypeStatistics defaultStatistic;
 
+    @Valid
+    @Nullable
+    private EventTypeOptions options;
```

Comment:
I agree that is not the clearest name for such field. I think it is more about internal Kafka adjustments and other properties which are related to internals of Nakadi.